### PR TITLE
🔧 NeoInputitems

### PR DESCRIPTION
**Pull request by roiLeo**
This can be tested on `/rmrk/mint` where you can add/remove tags

part of #5852 

- rewritte vue composition api for `AttributeTagInput`
- create `NeoInputitems` in libs/ui
- deleted unused components

Screenshot 2023-06-15 at 10-33-50 KodaDot Low fees and low carbon minting KodaDot - NFT Market Explorer

Screenshot 2023-06-15 at 11-21-30 KodaDot Low fees and low carbon minting KodaDot - NFT Market Explorer

### CODE REVIEW

**Comment by roiLeo** [comment]
Component updated with autocomplete padding & hover

File: components/rmrk/Create/AttributeTagInput.vue [hunk 1]
```diff
@@ -1,59 +1,61 @@
 <template>
-  <NeoField :label="$t('Tags')">
-    <b-taginput
+  <NeoField :label="$t('Tags')" :addons="false">
+    <NeoInputitems
```
Comment (vikiival) [inline components/rmrk/Create/AttributeTagInput.vue:3]:
```suggestion
    <NeoTagInput?
```

**Review by vikiival — COMMENTED** [review]
Naming otherwise LGTM

File: components/rmrk/Create/AttributeTagInput.vue (hunk 1 above)
Comment (roiLeo) [reply components/rmrk/Create/AttributeTagInput.vue:3]: Well Yes and no, more easy to find documentation about `InputItems` on oruga than `TagInput` which doesn't exist

Comment (roiLeo) [reply components/rmrk/Create/AttributeTagInput.vue:3]: lmk if you are sure about this one

**Comment by prury** [comment]
seems to be working fine, is there any place i can check those tag after i mint the collection?

**Comment by roiLeo** [comment]
> seems to be working fine, is there any place i can check those tags after i mint it?

What if I told you it's purely
magical-magic

**Review by vikiival — APPROVED** [review]

**Pull request merged**